src/xcode/ENA/ENA/Source/Services/HealthCertificate/Models/HealthCertificate.swift

```diff
@@ -87,6 +93,17 @@ struct HealthCertificate: Codable, Equatable, Comparable {
 	}
 
 	var type: CertificateType {
```

Comment:
Whats the reason for having type? Cant we just only use entry? If you read the entry and you don't need the associated value, you can omit it.

Reply:
With associated value I couldn't just write e.g. `$0.entry == .test`, as the default `Equatable` implementation always requires an associated value to equate to. And the `if case .test = $0.entry` variant is very inconvenient for `.filter`ing etc.

Reply:
Ok, I unterstand!